src/parsing/processProlog.ts

```diff
@@ -89,6 +90,37 @@ export default function processProlog(
 		staticContext.registerNamespace(prefix, namespaceURI);
 	});
 
+	// Default function namespace declaration
+	const defaultNamespaceFunctionDecl = astHelper
+		.getChildren(prolog, 'defaultNamespaceDecl')
+		.filter(x => {
+			return (
+				astHelper.getTextContent(astHelper.getFirstChild(x, 'defaultNamespaceCategory')) ===
```

Comment:
Could you add a not implemented error right here if the defaultNamespaceCategory is NOT equal to function? We still do not support the `default element namespace` syntax. We now 'accept' it and behave incorrectly. I would rather have an explicit 'NOT IMPLEMENTED' error here.

Reply:
done